utils/auth/src/lib.rs

```diff
@@ -64,6 +64,7 @@ pub fn create_token(perms: Vec<String>, key: &[u8]) -> JWTResult<String> {
 pub fn verify_token(token: &str, key: &[u8]) -> JWTResult<Vec<String>> {
     let mut validation = jsonwebtoken::Validation::new(jsonwebtoken::Algorithm::default());
     validation.validate_exp = false;
+    validation.required_spec_claims.remove("exp");
```

Comment:
I'm not sure it shouldn't be the other way around, i.e. adding this expiration to the token creation and validating it. Thoughts?

Reply:
I think adding `exp` to token payload is a better way to handle this
but right now admin token is printed out to stdout when daemon starts.
I am not sure if we have an option to request for a new admin token
another way would be to print out a new admin token once the previous one expires 

also what should be the expiration time for this token?

Reply:
We can generate a new admin token every time we start `forest`. To my understanding, it should not invalidate the previous ones. Only the signature and the expiration time would be different, but they would all be signed by the same private key.

Expiration time could be, e.g. 60 days, optimally configurable via a config flag. We can do it all in a separate task. If you want to proceed this way, could you please create an issue to implement proper expiration handling (and creation)?

Reply:
yeah sounds good
I'll create a new issue for this and work on it

Reply:
Alright, please link it here and then we can merge this one.